ui/ops/src/routes/ops/reports/Reports.vue

```diff
@@ -0,0 +1,95 @@
+<script setup>
+import {timestampToDate} from '@/api/convpb.js';
+import {getDownloadReportUrl, listReports} from '@/api/ui/reports.js';
+import ContentCard from '@/components/ContentCard.vue';
+import {useCohortStore} from '@/stores/cohort.js';
+import {useUiConfigStore} from '@/stores/uiConfig.js';
+import {computed, onMounted, ref} from 'vue';
+
+const reports = ref([]);
+
+const props = defineProps({
+  source: {
+    type: String,
+    default: ''
+  }
+});
+const uiConfig = useUiConfigStore();
+const cohort = useCohortStore();
+const name = computed(() => props.source || (uiConfig.config?.ops?.reports?.source ?? cohort.hubNode?.name ?? ''));
```

Comment:
If there's no fixed nav, then there should be no fixed config: remove the `ops.reports.source` and `cohort.hubNode.name` options here please